cli/ingress-controller/main.go

```diff
@@ -413,10 +413,24 @@ func main() {
 	cacheStores.Plugin = kongPluginInformer.GetStore()
 	informers = append(informers, kongPluginInformer)
 
-	kongClusterPluginInformer := kongInformerFactory.Configuration().V1().KongClusterPlugins().Informer()
-	kongClusterPluginInformer.AddEventHandler(reh)
-	cacheStores.ClusterPlugin = kongClusterPluginInformer.GetStore()
-	informers = append(informers, kongClusterPluginInformer)
+	// iterating through all our CRDs to find one seems like overkill, but it doesn't seem like there's
+	// a client function to check support for a single resource only. Alternatively we could try to infer this
+	// if cliConfig.WatchNamespace is set, since that precludes seeting any KongClusterPlugins
```

Comment:
It is best to keep such types of comments in the Git commit message rather than in code.